src/components/Avatar/styled/image.ts

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components/native';
+import { BaseProps } from '../../types';
+
+interface Props extends BaseProps {
```

Comment:
No need to extend from BaseProps since this component doesn't receive a style prop.